examples/codesandbox/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "react",
```

Comment:
Can we just move everything to the example folder. There is no need to have a separate folder for codesandbox

Reply:
Done.